src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -56,12 +55,68 @@ void createStation() {
      * Then 2개의 지하철역을 응답 받는다
      */
     // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록 조회")
+    @Test
+    void getStations() {
+        String gangNamStation = "강남역";
+        String songPaStation = "송파역";
+        createStation(gangNamStation);
+        createStation(songPaStation);
+
+        JsonPath stationJsonPath = findStations();
+
+        assertThat(stationJsonPath.get("[0].name").toString()).isEqualTo(gangNamStation);
+        assertThat(stationJsonPath.get("[1].name").toString()).isEqualTo(songPaStation);
```

Comment:
타입 변환을 해주는 메서드도 있습니다 ㅎㅎ

```suggestion
        assertThat(stationJsonPath.getString("[0].name")).isEqualTo(gangNamStation);
        assertThat(stationJsonPath.getString("[1].name")).isEqualTo(songPaStation);
```

Reply:
> 인수 테스트 중 지하철역 목록 조회시 응답값을 List 객체로 만드는것이 아닌
JsonPath의 값으로 이름 존재 유무를 확인해 보았는데 어떤것이 더 효율적인지 감이 오질 않네요.
피드백 주시면 감사하겠습니다! 🙏

개인적으로는 `ExtractableResponse<Response>`를 사용하는 편이기도하고,
테스트를 위해 같은 행위를 하는 것이기 때문에 선택의 차이 같아서 딱히 뭐가 효율적이다라고 판단을 하기가 좀 애매할 것 같습니다 🤔 

혹시 JsonPath를 사용하신 특별한 이유는 있으실까요?